config/initializers/callback_terminator.rb

```diff
@@ -1,6 +0,0 @@
-# Be sure to restart your server when you modify this file.
-
-# Do not halt callback chains when a callback returns false. This is a new
-# Rails 5.0 default, so it is introduced as a configuration option to ensure
-# that apps made with earlier versions of Rails are not affected when upgrading.
-ActiveSupport.halt_callback_chains_on_return_false = false
```

Comment:
@mkanoor - Was this change made to remove the Deprecated warnings only? Or was there a bug you were running into?

Reply:
@syncrou When I run the spec locally I get the deprecation warning